packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -857,6 +857,13 @@ class RenderBackdropFilter extends RenderProxyBox {
 
 /// A class that provides custom clips.
 abstract class CustomClipper<T> {
+  /// Creates a custom clipper.
+  ///
+  /// The clipper will update its clip whenever [reclip] notifies its listeners.
+  const CustomClipper({ Listenable reclip }) : _reclip = reclip;
+
+  final Listenable _reclip;
```

Comment:
I think you should discuss reclip in the class description dartdocs (also maybe describe the general design of the class there), and in more detail in the constructor description.